index.html

```diff
@@ -116,5 +116,68 @@ <h3 class="course-thumbnail-name">solo</h3>
         </a>
       </div>
     </section>
+    <section id="application" class="application">
+      <div class="application-container new-container">
+        <h2 class="application-section-head">Let't Try!</h2>
+        <p class="section-body-application">
+          Grand
+          Playersでは、ご利用登録いただけた方に対し、３レッスン分の無料枠をプレゼントいたします！
+          <br />
+          今すぐレッスンを初めて、”Grand”なプレイヤーを目指しましょう！
+          <br />
+          お申し込みは下記フォームからどうぞ！
+        </p>
+        <f<div class="application-input-container">
```

Comment:
```suggestion
        <form style="width: inherit">
         <div class="application-input-container">
```
<f となっているのはタイピングミスかな...？
「width: inherit」は親要素の内容を子要素に継承してくれる便利なCSSです！

以下参考サイトです
[https://developer.mozilla.org/ja/docs/Web/CSS/inherit](url)
[https://web-camp.io/magazine/archives/104553](url)

Reply:
コメントありがとうございます。 
<form style="width: inherit">を打ち込んでも画面幅が変わりませんでした。